packages/twenty-front/src/modules/settings/data-model/fields/forms/components/SettingsObjectNewFieldSelector.tsx

```diff
@@ -128,7 +130,15 @@ export const SettingsObjectNewFieldSelector = ({
                     .map(([key, config]) => (
                       <StyledCardContainer key={key}>
                         <UndecoratedLink
-                          to={`/settings/objects/${objectNamePlural}/new-field/configure?fieldType=${key}`}
+                          to={getSettingsPath(
+                            SettingsPath.ObjectNewFieldConfigure,
+                            {
```

Comment:
neat: 
```
{ 
  xxx,
}
```
is valid but
`{  xxx }`
is also valid and shorter

Reply:
OK! Changed more items in the codebase: https://github.com/twentyhq/twenty/pull/9735/commits/295299651bdad162ba608356473b38496c8dbb4c